strawberryfields/circuitspecs/gbs.py

```diff
@@ -52,6 +52,11 @@ class GBSSpecs(GaussianSpecs):
         "Fouriergate",
```

Comment:
```suggestion
```

Reply:
How come this circuit specification doesn't allow S2gates? 🤔

Reply:
That is a great question!

Reply:
![image](https://user-images.githubusercontent.com/991946/68229409-3ab39a80-ffc5-11e9-8591-ea1e6e4a9d7f.png)

Reply:
I added the S2gate into the gbs circuit spec and seems like things work except that the test coverage plummets which is very weird.